### ISSUE TITLE
[Feature request] Undo review in Anki mode

### ISSUE THREAD

**Issue by cpuchi** [davidsansome/tsurukame#788]
It would be great to have an undo option when using Anki mode (like when using the actual Anki software). Sometimes I get too overzealous on the button and it would be awesome to undo

**Comment by WyattLuke** [comment]
Yeah, I can concur on this, sometimes I think I know something, don't check, and hit correct, only to then notice I got it wrong. getting an undo would be great.
